common/shield_exceptions.cc

```diff
@@ -28,60 +26,4 @@ bool IsUAWhitelisted(const GURL& gurl) {
       });
 }
 
-bool IsBlockedResource(const GURL& gurl) {
-  static std::vector<URLPattern> blocked_patterns({
-      URLPattern(URLPattern::SCHEME_ALL, "https://pdfjs.robwu.nl/*")
-  });
-  return std::any_of(blocked_patterns.begin(), blocked_patterns.end(),
-                     [&gurl](URLPattern pattern){
-                       return pattern.MatchesURL(gurl);
-                     });
-}
-
-bool IsWhitelistedFingerprintingException(const GURL& firstPartyOrigin,
-    const GURL& subresourceUrl) {
-  // Always allow embeds from public.tableau.com while fingerprinting
-  // protections are being reworked to need less exceptions.
-  static const std::vector<URLPattern> embed_exceptions = {
-    URLPattern(URLPattern::SCHEME_ALL, "https://public.tableau.com/*"),
-    URLPattern(URLPattern::SCHEME_ALL, "https://www.arcgis.com/*"),
-  };
-  for (const auto& exception : embed_exceptions) {
-    if (exception.MatchesURL(subresourceUrl)) {
-      return true;
-    }
-  }
-
-  static std::map<URLPattern, std::vector<URLPattern> > whitelist_patterns = {
```

Comment:
I think you can now remove `#include <map>`

Reply:
Thanks @iefremov, just removed :)